config/software/bzip2.rb

```diff
@@ -40,7 +40,7 @@
   args = "PREFIX='#{install_dir}/embedded' VERSION='#{version}'"
 
   patch source: 'makefile_take_env_vars.patch', env: env
-  patch source: 'soname_install_dir.patch', env: env if mac_os_x_mavericks?
+  patch source: 'soname_install_dir.patch', env: env if mac_os_x?
```

Comment:
Good point. Per our support policy for this repo, we only intend to support what we need to build Chef products, and per https://github.com/chef/chef-rfc/commit/970f0c80b51d64ad3132538b2a731382d9cba438 we only support Mavericks and up. So I think this is okay.